client/extensions/woocommerce/components/dashboard-widget/index.js

```diff
@@ -0,0 +1,142 @@
+/**
+ *
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Gridicon from 'gridicons';
+import { isUndefined, noop } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Popover from 'components/popover';
+import Tooltip from 'components/tooltip';
+
+class DashboardWidget extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			showDialog: false,
+			showTooltip: false,
+		};
+	}
+
+	hideTooltip = () => {
+		this.setState( { showTooltip: false } );
+	};
+
+	onSettingsPanelClose = () => {
+		this.setState( { showDialog: false } );
+	};
+
+	setSettingsToggle = c => {
+		this.settingsToggle = c;
+	};
+
+	showTooltip = () => {
+		this.setState( { showTooltip: true } );
+	};
+
+	toggleSettingsPanel = () => {
+		this.props.onSettingsClose();
+		this.setState( { showDialog: ! this.state.showDialog } );
+	};
+
+	render() {
+		const {
+			children,
+			className,
+			image,
+			imageFlush,
+			imagePosition,
+			settingsPanel,
+			title,
+			translate,
+			width,
+		} = this.props;
+		const { showDialog, showTooltip } = this.state;
+		const isTopImage = image && 'top' === imagePosition;
+		const imageClassName = image ? `is-${ imagePosition }` : null;
+		const widthClassName = `is-${ width }-width`;
+		const hasSettingsPanel = ! isUndefined( settingsPanel );
+
+		const classes = classNames( 'dashboard-widget', className, imageClassName, widthClassName, {
+			'is-flush-image': imageFlush,
+			'has-settings-panel': hasSettingsPanel,
+		} );
+
+		// eslint-disable-next-line wpcalypso/jsx-classname-namespace
```

Comment:
What's wrong with the class name? It looks like it's prefixed correctly.

Reply:
Right!? I honestly do not know why the linter kept freaking out about this particular line. I just opted to stop fighting and add the comment instead.